devtools-panels/devtools/panel/devtools-panel.js

```diff
@@ -0,0 +1,71 @@
+/**
+Handle errors from the injected script.
+Errors may come from evaluating the JavaScript itself
+or from the devtools framework.
+See https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/devtools.inspectedWindow/eval#Return_value
+*/
+function handleError(error) {
+  if (error.isError) {
+    console.log(`Devtools error: ${error.code}`);
+  } else {
+    console.log(`JavaScript error: ${error.value}`);
+  }
+}
+
+/**
+Handle the result of evaluating the script.
+If there was an error, call handleError.
+*/
+function handleResult(result) {
+  console.log('Function handleResult');
+  console.log(result);
+  if (result[0] !== undefined) {
+    console.log(`jQuery: ${result[0]}`);
```

Comment:
This issue still isn't addressed. Perhaps you could rename this function to handlejQueryTest, or something, then for the others, use an anonymous function, like:

  ```
browser.devtools.inspectedWindow.eval(evalString)
    .then(result => {
      if (result[1]) {
        handleError(result[1]);
      }
    });
```